examples/docking-protein-peptide/docking-protein-peptide-mdref-full.cfg

```diff
@@ -0,0 +1,81 @@
+# ====================================================================
+# Protein-peptide docking example 
+
+# directory in which the scoring will be done
+run_dir = "run1-mdref-full"
+
+# ###
+mode = "hpc"
+# in which queue the jobs should run, if nothing is defined
+#  it will take the system's default
+# queue = "short"
+# concatenate models inside each job, concat = 5 each .job will produce 5 models
+concat = 5
+#  Limit the number of concurrent submissions to the queue
+queue_limit = 100
+# cns_exec = "path/to/bin/cns" # optional
+# ###
+# Rigid-body docking example
+
+# molecules to be docked
+molecules =  [
+    "data/1NX1_protein.pdb",
+    "data/DAIDALSSDFT_3conformations.pdb"
+    ]
+
+# ====================================================================
+# Parameters for each stage are defined below, prefer full paths
+#####################################################################
+# WARNING: THE PARAMETERS HERE ARE ILLUSTRATIVE
+#  THE WORKFLOW IS WORK-IN-PROGRESS
```

Comment:
I guess its no longer a work-in-progress?

Reply:
Indeed - just pushed an update that removes those.